### PR TITLE
Alexa to the rescue of inaccessible websites.

### PR DESCRIPTION
**Pull request by alexandrenicol**
This PR contains the post for my upcoming talk about my side project that uses Alexa to make inaccessible websites accessible.

I've tried to follow the guides [here](https://github.com/asyncjs/async-website/wiki/Adding-an-event) and [there](https://github.com/asyncjs/async-website/blob/master/_posts/0000-01-01-_blank.md) as much as possible, and took some additional things from previous posts like the links to the meetup pages and co.

I wasn't actually sure where will the event be? Spaces?

Thanks!

### CODE REVIEW

File: _posts/2023-02-02-alexa-rescues-inaccessible-sites.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+online: true
+---
+
+The Web is over 30 years old, yet some websites are still not accessible.
+
+This is the tale of how Alexa, with a little bit of help, came to the rescue of a blind person who could not get access to local news.
+
+In this talk, Alex Nicol speaks about accessibility, web crawling, tech-for-good, Alexa, running Ruby on Lambdas, and working on a side project whilst having a six-month-old baby.
+
+Alex Nicol is a web developer working at [BridgeU](https://bridge-u.com/) and [Bounce Technologies](https://www.bouncetechnologies.com/). He's also founded is [own consultancy company](https://alexnicol.dev), specialised in conversational systems and tech-for-good. He previously spent six years at EDF Energy as an R&D engineer specialising in automated voice systems and chatbots.
```
Comment (JakeSidSmith) [inline _posts/2023-02-02-alexa-rescues-inaccessible-sites.md:34]:
```suggestion
Alex Nicol is a web developer working at [BridgeU](https://bridge-u.com/) and [Bounce Technologies](https://www.bouncetechnologies.com/). He's also founded his [own consultancy company](https://alexnicol.dev), specialised in conversational systems and tech-for-good. He previously spent six years at EDF Energy as an R&D engineer specialising in automated voice systems and chatbots.
```

File: _posts/2023-02-02-alexa-rescues-inaccessible-sites.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  title: "HAL in vector by abelmon007 is licensed under CC BY 2.0."
+  link:  https://www.flickr.com/photos/80265839@N00/4682461024
+tags:
+- alexa
+- accessibility
+- ruby
+- aws
+- crawler
+venue:
+  name: TBC?
```
Comment (JakeSidSmith) [inline _posts/2023-02-02-alexa-rescues-inaccessible-sites.md:21]:
```suggestion
venue:
  name: Spaces
  link: https://www.spacesworks.com/brighton/trafalgar-place/
  location: https://g.page/Mocatta-House-4437?share
  address: Mocatta House, Trafalgar Pl, BN1 4DU, Brighton
  latlong: 50.8294399,-0.1443907
```

File: _posts/2023-02-02-alexa-rescues-inaccessible-sites.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Alexa to the rescue of inaccessible websites"
+summary: "This is the tale of how Alexa, with a little bit of help, came to the rescue of a blind person who could not get access to local news."
+date: 2023-02-02T19:15:00
+meetup: https://www.meetup.com/Async-Web-Tech-Meetup/
```
Comment (JakeSidSmith) [inline _posts/2023-02-02-alexa-rescues-inaccessible-sites.md:5]: Note to self to add meetup link.

File: _posts/2023-02-02-alexa-rescues-inaccessible-sites.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+This is the tale of how Alexa, with a little bit of help, came to the rescue of a blind person who could not get access to local news.
+
+In this talk, Alex Nicol speaks about accessibility, web crawling, tech-for-good, Alexa, running Ruby on Lambdas, and working on a side project whilst having a six-month-old baby.
+
+Alex Nicol is a web developer working at [BridgeU](https://bridge-u.com/) and [Bounce Technologies](https://www.bouncetechnologies.com/). He's also founded is [own consultancy company](https://alexnicol.dev), specialised in conversational systems and tech-for-good. He previously spent six years at EDF Energy as an R&D engineer specialising in automated voice systems and chatbots.
+
+You will be able to join us in-person at [Spaces](https://www.spacesworks.com/brighton/trafalgar-place/), or online (via Zoom).
+
+If you're thinking of joining us in-person, please sign up to the event on [meetup.com](https://www.meetup.com/async-web-tech-meetup/events/288819953/). We also highly recommend you sign up to the [Async Slack](https://join.slack.com/t/asyncjs/shared_invite/zt-1aguxx86q-XjF_yWcFoJ8fyYYzoqgDaQ) for more info, and updates.
```
Comment (JakeSidSmith) [inline _posts/2023-02-02-alexa-rescues-inaccessible-sites.md:38]: Note to self to add meetup link.

**Review by JakeSidSmith — COMMENTED** [review]
Yes, we'll be at Spaces. Have added suggestion for the venue details. 😊 
The rest are notes for myself/whomever sets up the Meetup.

**Comment by alexandrenicol** [comment]
@JakeSidSmith Hey, is there anything I can do to help with this?

**Comment by JakeSidSmith** [comment]
> @JakeSidSmith Hey, is there anything I can do to help with this?

No, sorry, we're just waiting on some other stuff. 😊

**Pull request merged**